test/functional/test_framework/test_node.py

```diff
@@ -673,6 +673,7 @@ def add_p2p_connection(self, p2p_conn, *, wait_for_verack=True, **kwargs):
     def add_outbound_p2p_connection(self, p2p_conn, *, wait_for_verack=True, p2p_idx, connection_type="outbound-full-relay", **kwargs):
         """Add an outbound p2p connection from node. Must be an
         "outbound-full-relay", "block-relay-only", "addr-fetch" or "feeler" connection.
+        The txrelay arg determines whether our peer will signal support for txrelay in their version message.
```

Comment:
831ed4960bfb1eafb763cb2a52201a97a17f8168: seems like this should be in the `peer_connect_send_version` function docstring?

Reply:
Removed the doc, since it's hidden in the `kwargs` anyway (It wasn't when I added it in an earlier revision). Within `peer_connect_send_version` it's kind of self-explanatory, I had added it here to avoid confusion to which side of the connection `txrelay` refers to.